### PR TITLE
#28947 Fix scope reset on new synonym form

### LINKED ISSUE #28947: Scope not reset on New Synonym Group form
Scope not reset on New Synonym Group form

### Preconditions (*)
Magento 2.3.4

### Steps to reproduce (*)

1. Admin - Marketing - SEO & Search 
2. Click on New Synonym Group
3. Select scope different then default 
4. Enter other details 
5. Click on Reset 

### Expected result (*)
All values should be reset 

### Actual result (*)
image

### PR DESCRIPTION
**Pull request by srsathish92**
### Description (*)
This PR to update correct default value "0:0" & data type for scope field in new synonym group form. that's why reset not working due to incorrect default value on new form.

### Related Pull Requests
https://github.com/magento/magento2/pull/29193 - I guess this js change not required because rest of the select resetting works fine in other forms (ex: customer form)

### Fixed Issues (if relevant)
1. Fixes magento/magento2#28947

### Manual testing scenarios (*)
1. please check steps to reproduce on #28947

### Questions or comments
NA

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by srsathish92** [comment]
@magento run all test

**Comment by rogyar** [comment]
Hi @srsathish92. Good catch, thank you. Due to the [definition of done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all new changes should be covered by auto-tests. Could you write an MFTF test that goes through the "Steps to reproduce" and makes sure that the issue does not appear anymore? 

Thank you!

**Comment by srsathish92** [comment]
@magento run all tests

**Comment by srsathish92** [comment]
@rogyar I have updated MFTF test case. Can you please review now?

**Comment by srsathish92** [comment]
@magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B,Sample Data Tests CE,Sample Data Tests EE,Sample Data Tests B2B

File: app/code/Magento/Search/Test/Mftf/Test/AdminNewSearchSynonymsFormResetTest.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <group value="Search"/>
+        </annotations>
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="LoginAsAdmin"/>
+        </before>
+        <after>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <!--TEST BODY -->
```
Comment (rogyar) [inline app/code/Magento/Search/Test/Mftf/Test/AdminNewSearchSynonymsFormResetTest.xml]: Please, remove comments from the test body. In this particular case, the comments don't provide important information but will be picked up by a test runner as separate test steps.

**Comment by rogyar** [comment]
Hi @srsathish92. Good job. 
However, something went wrong with the functional tests. As you can see, all of them are failing with no report. Usually, it happens if the test generation step is broken somehow (a wrong reference, missing action group, etc). Please, try running the tests locally one more time.

**Comment by srsathish92** [comment]
@magento run all tests

File: app/code/Magento/Search/Test/Mftf/Test/AdminNewSearchSynonymsFormResetTest.xml (hunk 1 above)
Comment (srsathish92) [reply app/code/Magento/Search/Test/Mftf/Test/AdminNewSearchSynonymsFormResetTest.xml]: @rogyar I have removed comments. can you please review now?

**Comment by srsathish92** [comment]
@magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B, Integration Tests

**Comment by srsathish92** [comment]
@magento run Functional Tests B2B, Integration Tests

**Review by rogyar — APPROVED** [review]
The same failing integration test occurs for other PRs, so we may assume that it's not related. RMA functionality is not affected by the current changeset somehow, so the failing functional tests are not related either. 

@srsathish92 great job!

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-7917](https://jira.corp.magento.com/browse/ENGCOM-7917) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Manual testing scenario:**

1. Go to Admin -> Marketing -> SEO & Search -> Search Synonyms;
2. Click on **New Synonym Group** button;
3. Select scope different than the default;
4. Enter other details;
5. Click on **Reset** button;

**Before:** :heavy_multiplication_x:  Not all items have been reset

Peek 2020-07-31 10-09

**After:** :heavy_check_mark: All items have been reset

Peek 2020-07-31 10-32

**Comment by engcom-Echo** [comment]
@magento run all tests

**Pull request merged**